src/EPR.Calculator.API/Services/TransposePomAndOrgDataService.cs

```diff
@@ -130,5 +157,56 @@ public void Transpose(CalcResultsRequestDto resultsRequestDto)
                 }
             }
         }
+
+        private List<OrganisationDetails> GetOrganisationDetailsBySubmissionPeriod(int runId)
```

Comment:
Return can be IEnumerable<OrganisationDetails>

Reply:
updated the return type.